schema/m3-requirements.schema.json

```diff
@@ -725,7 +725,7 @@
             "properties": {
               "frames": {
                 "type": "integer",
-                "minimum": 1,
+                "minimum": 0,
```

Comment:
The Rmode case tries to be a 0 cost shinespark so I could see not trying to write it as 1 frame with 1 excess.